out-adapter-persistence-jpa/src/main/java/gohigher/application/entity/ApplicationJpaEntity.java

```diff
@@ -97,6 +97,27 @@ public void addProcess(ApplicationProcessJpaEntity process) {
 		processes.add(process);
 	}
 
+	public void changeToDelete() {
+		this.deleted = true;
+	}
+
+	public Application toDomain() {
+		List<Process> processes = this.processes.stream()
+			.sorted(Comparator.comparingInt(ApplicationProcessJpaEntity::getOrder))
+			.map(ApplicationProcessJpaEntity::toDomain)
+			.toList();
+
+		if (currentProcessOrder == 0) {
+			return new Application(id, companyName, team, location, contact, duty, position, jobDescription, workType,
+				employmentType, careerRequirement, requiredCapability, preferredQualification, deadline, processes, url,
+				null);
+		}
+
+		return new Application(id, companyName, team, location, contact, duty, position, jobDescription, workType,
+			employmentType, careerRequirement, requiredCapability, preferredQualification, deadline, processes, url,
+			processes.get(currentProcessOrder));
+	}
+
 	public Application toCalenderDomain() {
 		List<Process> processes = this.processes.stream()
 			.sorted(Comparator.comparingInt(ApplicationProcessJpaEntity::getOrder))
```

Comment:
```suggestion
	public Application toCalendarDomain() {
		List<Process> processes = this.processes.stream()
			.sorted(Comparator.comparingInt(ApplicationProcessJpaEntity::getOrder))
```
위의 `toDomain()` 에 있는 로직과 동일해서 메서드를 추출할 수 있을것 같아요!

Reply:
해당 부분은 currentProcess를 관리하는 정책이 확실히 정해지지 않아서 현재 중복이 발생하였다 생각합니다!
currentProcess의 정책을 확실히 정한다면 해당 중복도 제거하고 둘은 완벽하게 서로 다른 메서드가 될 것이라 생각합니다!

그리고 `toCalendarDomain()`은 반환값이 동석님이 말씀해주신 `SimpleApplication` 객체로 변경될 것이라 생각하여 코드의 중복이어도 추출을 하지 않았습니다!